build.sbt

```diff
@@ -12,8 +12,16 @@ developers in ThisBuild := List(
 )
 homepage in ThisBuild := Some(url("http://scalamock.org/"))
 
-lazy val scalatest = "org.scalatest" %% "scalatest" % "3.0.3"
-lazy val specs2 = "org.specs2" %% "specs2-core" % "3.9.1"
+lazy val scalatest = "org.scalatest" %% "scalatest" % "3.0.5-M1"
+lazy val specs2 = Def.setting {
+  val v = CrossVersion.partialVersion(scalaVersion.value) match {
+    case Some((2, 10)) =>
+      "3.9.1" // specs2 4.x does not support Scala 2.10
```

Comment:
Are 3.9.x and 4.0.x guaranteed to be API compatible?

Reply:
I think so